src/data/faq_de.json

```
@@ -376,6 +376,217 @@
+                                    "Es handelt sich hierbei um Vorkehrungen, die getroffen worden sind, um Missbrauch dieser Funktion zu vermeiden. Beachten Sie, dass Sie mit positiven Testergebnissen, die Sie in Ihrer App erhalten, trotzdem warnen können."
+                                ]
+                            },
+                            {
+                                "title": "Werden trotz der neuen Funktion die Ergebnisse auch weiterhin digital in die App übertragen?",
+                                "anchor": "warn_without_tan_digital_result",
+                                "active": false,
+                                "textblock": [
+                                    "Selbstverständlich; sofern die Teststelle bzw. das beauftragte Labor technisch an die Infrastruktur angebunden sind, können auch weiterhin die Ergebnisse der Nachweistests digital für die Corona-Warn-App bereitgestellt werden. Für die Anwender*innen der CWA ändert sich diese Funktion nicht. Das bisherige Verfahren wird durch die neue Funktion lediglich ergänzt, um mit allen Testarten warnen und damit noch mehr Kontaktketten zeitnah unterbrechen zu können."
+                                ]
+                            },
+                            {
+                                "title": "Ich habe mein positives Ergebnis aus anderem Wege erfahren (E-Mail, Telefonat…), aber kein Ergebnis in der App – kann ich trotzdem meine Kontakte warnen?",
+                                "anchor": "warn_without_tan_external_result",
+                                "active": false,
+                                "textblock": [
+                                    "Ja, nutzen Sie ab Release 3.0 die neue Funktion \"Tests erfassen und andere warnen\", dazu ist ggf. ein Update der App nötig.",
+                                    "Warnen Sie damit Andere auf Basis des empfangenen positiven Ergebnisses manuell in der CWA.",
+                                    "Siehe: <a href='placeholder' target='_blank'>Wie kann ich andere Nutzer warnen, wenn ich positiv getestet worden bin?</a>"
+                                ]
+                            },
+                            {
+                                "title": "Muss ich trotzdem die Hotline anrufen?",
+                                "anchor": "warn_without_tan_hotline",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, ab Release 3.0 der Corona-Warn-App ist es nicht mehr notwendig, die Verifikations-Hotline anzurufen."
+                                ]
+                            },
+                            {
+                                "title": "Muss ich auf das neue Release umsteigen, oder läuft meine aktuelle Version der CWA weiter?",

```

1. Wording changed to make it less complex to maintain (there will be a time when the feature/version isn't new anymore)
2. Changed "Release" to "Version" 

```suggestion
                                "title": "Muss ich auf Version 3.0 umsteigen, oder läuft meine aktuelle Version der CWA weiter?",
```

---

Your suggestion has been comitted, thanks